stories/buttons/button-story.js

```diff
@@ -74,68 +74,68 @@ const ButtonStory = ({ variant }) => {
       <div className='flex flex-row'>
         <div className='mr-20'>
           <p className='text-primary-700 mt-20 mb-3'>GROUPED (horizontal):</p>
-          <ButtonGroup variant={variant}>
-            <Button variant={variant} size='lg'><Add size='lg' /></Button>
-            <Button variant={variant} size='lg'><Add size='lg' /></Button>
-            <Button variant={variant} size='lg'><Add size='lg' /></Button>
-            <Button variant={variant} size='lg'><Add size='lg' /></Button>
+          <ButtonGroup variant={variant} size='lg'>
+            <Button><Add size='lg' /></Button>
+            <Button><Add size='lg' /></Button>
+            <Button><Add size='lg' /></Button>
+            <Button><Add size='lg' /></Button>
           </ButtonGroup>
           <div className='mt-5'>
-            <ButtonGroup variant={variant}>
-              <Button variant={variant} size='lg'>Button</Button>
-              <Button variant={variant} size='lg'>Button</Button>
-              <Button variant={variant} size='lg'>Button</Button>
-              <Button variant={variant} size='lg'><Add size='lg' /></Button>
+            <ButtonGroup variant={variant} size='lg'>
+              <Button>Button</Button>
+              <Button>Button</Button>
+              <Button>Button</Button>
+              <Button><Add size='lg' /></Button>
             </ButtonGroup>
           </div>
           <div className='mt-5'>
-            <ButtonGroup variant={variant}>
-              <Button variant={variant} size='md'>Button</Button>
-              <Button variant={variant} size='md'>Button</Button>
-              <Button variant={variant} size='md'>Button</Button>
-              <Button variant={variant} size='md'><Add size='md' /></Button>
+            <ButtonGroup variant={variant} size='md'>
+              <Button size='md'>Button</Button>
```

Comment:
We want to delete also the size prop in Button components in this group as well. Right?

Reply:
whoops, too many missed that one lol, fixed!